bin/send_email.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import sys
+import synapseclient
+
+
+def get_participant_id(syn, submission_id):
+    """
+    Retrieves the teamId of the participating team that made
+    the submission. If the submitter is an individual rather than
+    a team, the userId for the individual is retrieved.
+    """
+    # Retrieve a Submission object
+    submission = syn.getSubmission(submission_id, downloadFile=False)
+
+    # Get the teamId or userId of submitter
+    participant_id = submission.get("teamId") or submission.get("userId")
+
+    # Ensure that the participant_id returned is a list
+    # so it can be fed into syn.sendMessage(...) later.
+    return [participant_id]
+
+
+def send_email(view_id, submission_id):
+    """
+    Sends an e-mail on the status of the individual submission
+    to the participant team or participant individual.
+    """
+    syn = synapseclient.login()
+    status = syn.getSubmissionStatus(submission_id)["submissionAnnotations"]["validation_status"]
+
+    # Get the synapse users to send an e-mail to
+    ids_to_notify = get_participant_id(syn, submission_id)
+
+    # Sends an e-mail notifying participant(s) that the evaluation succeeded
+    if status == ["VALIDATED"]:
```

Comment:
I'm a little confused as to how this is working right now, if `SEND_EMAIL` runs after `UPDATE_SUBMISSION_STATUS_AFTER_SCORE`, then the status would be `SCORED`, wouldn't it?

Reply:
Oh nevermind I misread the `getSubmissionStatus` line

Reply:
The status is the `validation_status`, which is either `INVALID` or `VALIDATED`. I can update the variable name to `validations_status` to make this clearer.